README.md

```diff
@@ -115,6 +115,8 @@ playground('.selector', options)
 
 - `onOpenConsole` — Is called after the console's opened.
 
+- `onOutputAddedToDom` - Is called after the output is added to DOM.
```

Comment:
Pls, more netrual, pls `onOutputUpdate` or `onOutputData` or smth like that.

Reply:
I fixed that